### PR TITLE
Proposed fix for TestOperationErrorReported.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: worker/uniter/uniter_test.go [hunk 1]
```diff
@@ -2009,7 +2009,7 @@ func (m *mockExecutor) Run(op operation.Operation) error {
 }
 
 func (s *UniterSuite) TestOperationErrorReported(c *gc.C) {
-	c.Skip("maltese-falcon")
+	//c.Skip("maltese-falcon")
```
Comment (mattyw) [inline worker/uniter/uniter_test.go]: remove this line

**Comment by mattyw** [comment]
LGTM

**Comment by tasdomas** [comment]
LGTM

**Comment by alesstimec** [comment]
$$merge$$

**Comment by alesstimec** [comment]
$$merge$$

**Comment by alesstimec** [comment]
$$merge$$

**Comment by alesstimec** [comment]
$$merge$$ $$merge$$ $$merge$$

**Comment by alesstimec** [comment]
please $$merge$$

**Comment by alesstimec** [comment]
$$merge$$

**Pull request merged**